lib/turn.rb

```diff
@@ -1,6 +1,38 @@
 class Turn
-  def initialize()
+  attr_reader :board, :player1, :player2
+  def initialize(board, player1, player2)
+    @board = board
+    @player1 = player1
+    @player2 = player2
   end
 
+# Method to have the computer select a random (and not full) column
+# and print the board.
 
+  def computer_move
+    move = @board.game_board.keys.sample
+    if draw?
```

Comment:
could we add the message to the method on L34? to help trim the computer_move method?